docs/standard/translation/understanding_transifex.md

```diff
@@ -11,6 +11,7 @@ Transifex is organized into projects:
 * [OCDS Extension Explorer](https://www.transifex.com/OpenDataServices/ocds-extension-explorer/dashboard/)
 * [OCDS Extensions](https://www.transifex.com/OpenDataServices/ocds-extensions/dashboard/)
 * [OCDS for PPPs](https://www.transifex.com/OpenDataServices/ocds-for-ppps/dashboard/)
+* [OC4IDS 9.0](https://www.transifex.com/OpenDataServices/oc4ids-09/dashboard/)
```

Comment:
Isn't it 0.9 not 9.0?

Reply:
Apologies, yes, I seem to have accelerated development a bit.
Now fixed.